MapboxNavigation/InstructionPresenter.swift

```diff
@@ -63,12 +65,12 @@ class InstructionPresenter {
         return attributedPairs.attributedStrings
     }
     
-    typealias AttributedInstructionComponents = (components: [VisualInstructionComponent], attributedStrings: [NSAttributedString])
+    typealias AttributedInstructionComponents = (components: [MBComponentRepresentable], attributedStrings: [NSAttributedString])
     
     func attributedPairs(for instruction: VisualInstruction, dataSource: DataSource, imageRepository: ImageRepository, onImageDownload: @escaping ImageDownloadCompletion) -> AttributedInstructionComponents {
-        let components = instruction.textComponents
+        let components = instruction.components.compactMap { $0 as? VisualInstructionComponent }
         var strings: [NSAttributedString] = []
-        var processedComponents: [VisualInstructionComponent] = []
+        var processedComponents: [MBComponentRepresentable] = []
```

Comment:
This can simply be `ComponentRepresentable`.

Reply:
@bsudekum it's a protocol. Hence the prefix `MB`

Reply:
I think the MB should be renamed from the swift name of this protocol: https://github.com/mapbox/MapboxDirections.swift/pull/258/files#diff-664678368965fd0cdce04411baf6644cR7